.github/workflows/upload-release-artifacts.yml

```
@@ -37,4 +37,12 @@ jobs:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: .gravitybee\dist\latest\aqt.exe
           tag: ${{ github.ref }}
-          overwrite: true
\ No newline at end of file
+          overwrite: true
+          
+      - name: Update continuous build
+        uses: marvinpinto/action-automatic-releases@latest
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          prerelease: true
+          automatic_release_tag: Continuous
+          file: .gravitybee\dist\latest\aqt.exe

```

Do we need to replace `\` with `/` here? Windows confuses me ...
```suggestion
          file: .gravitybee/dist/latest/aqt.exe

```

---

IIRC, using forward slashes is OK on Windows.